src/dht.cpp

```diff
@@ -941,8 +946,20 @@ Dht::searchStep(Search& sr)
     DHT_DEBUG("[search %s IPv%c] step", sr.id.toString().c_str(), sr.af == AF_INET ? '4' : '6');
     sr.step_time = now;
 
+    /*
+     * The accurate delay between two refills has not been strongly determined.
+     * TODO: Emprical analysis over refill timeout.
+     */
+    if (sr.refill_time + Node::NODE_EXPIRE_TIME < now and sr.nodes.size()-sr.getNumberOfBadNodes(now) < SEARCH_NODES) {
```

Comment:
We try to insert new nodes in the search every time we hear about them.
What is the case that we try to solve here ?
This will run every 10 minutes, how will it help in that scenario ?